test/langtools/tools/javac/diags/examples/NoTitle.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
It's surprising to see 2012 here.

Reply:
It was derived from a file created back then. But, it's no great less to lose the 2012.